doc/getting-started/apl-nuc.rst

```diff
@@ -113,6 +113,7 @@ manually, as described below).
    #.  Select "User Manager" to add an administrative user "clear" and
        password.
    #.  Select "Assign Hostname" to set the hostname as "clr-sos-guest"
+   #.  Select "Telemetry" to set Tab to highlight your choice.
```

Comment:
Changing the ``Telemetry`` option should be done before moving to the ``Advanced options``
``x.  Select "Telemetry" to set Tab to highlight your choice.``